packages/ui/src/helpers/logger/logger.ts

```diff
@@ -0,0 +1,62 @@
+type LoggerCategory =
+  | 'AccountSettings'
+  | 'Auth'
+  | 'Geo'
+  | 'Icon'
+  | 'Notifications'
+  | 'StateMachine'
+  | 'StorageImage'
+  | 'StorageManager';
+
+export enum LOG_LEVEL {
+  DEBUG = 'DEBUG',
+  ERROR = 'ERROR',
+  INFO = 'INFO',
+  WARN = 'WARN',
+}
+
+class Logger {
```

Comment:
Simplistic version of a Logger to cover our existing usecases, open to add more if needed.